example_test.go

```diff
@@ -0,0 +1,32 @@
+package scalewaysdkgo
+
+import (
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func Example_apiClient() {
```

Comment:
Let's keep things simple: no error handling, no pretty display.

Also we could add few comment on what does each line.

```Go
func Example_apiClient() {
	// create a Scaleway client
 	client, _ := scw.NewClient(
		scw.WithAuth("ACCESS_KEY", "SECRET_KEY"), // Get your credentials at https://console.scaleway.com/account/credentials
	)

	// get instance SDK
 	instanceApi := instance.NewApi(client)

	// do a request
 	response, _ := instanceApi.ListServers(&instance.ListServersRequest{
		Zone: scw.ZoneFrPar1,
	})

	fmt.Println(response)
 }
```

Reply:
I just copied the file from the `README.md` file.